grammars/regular expressions (python).cson

```diff
@@ -55,7 +56,7 @@
         'name': 'meta.assertion.look-behind.regexp'
       '6':
         'name': 'meta.assertion.negative-look-behind.regexp'
-    'end': '(\\))'
+    'end': '(\\))|(?="""|\'\'\')'
```

Comment:
I know :(
This has a chance of breaking valid regexes `(""")`, but hopefully that'll be much more rare than what this fixes (and that example should have a workaround `("{3}")`).